torch/_inductor/config.py

```diff
@@ -21,7 +21,9 @@ def bundled_autotune_remote_cache_default() -> Optional[bool]:
 
 
 def bundle_triton_into_fx_graph_cache_default() -> Optional[bool]:
-    return get_tristate_env("TORCHINDUCTOR_BUNDLE_TRITON_INTO_FX_GRAPH_CACHE")
+    return get_tristate_env(
+        "TORCHINDUCTOR_BUNDLE_TRITON_INTO_FX_GRAPH_CACHE", True if is_fbcode() else None
```

Comment:
Won' setting it to True in fbcode disallow the ability to disabled via JK?

Reply:
Good catch, meant to do no is_fbcode()